### PR TITLE
Various scpipe fixes

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/scpipe/scpipe.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@ ln -s '${exons}' '$anno_name' &&
 TAB=\$(printf '\t') &&
 
 #if $samples.barcodes:
-    sed -i.bak -e "s/\${TAB}/,/g" '$samples.barcodes' &&
+    sed -e "s/\${TAB}/,/g" '$samples.barcodes' > samples.barcodes &&
```
Comment (bernt-matthias) [inline tools/scpipe/scpipe.xml:77]: Nasty one. Good catch.

File: tools/scpipe/scpipe.xml [hunk 2]
```diff
@@ -244,8 +244,8 @@ Rscript '$__tool_directory__/scpipe.R'
         <test>
             <param name="format_select" value="fastq" />
             <param name="fasta_source" value="history"/>
-            <param name="ref_fa_hist" ftype="fasta" value="mm10_MT19.fa.gz"/>
-            <param name="exons" ftype="gff3" value="mm10_MT19.gff3.gz"/>
+            <param name="ref_fa_hist" ftype="fasta.gz" value="mm10_MT19.fa.gz"/>
```
Comment (bernt-matthias) [inline tools/scpipe/scpipe.xml:247]: Question: we do not need to list xyz.gz in the format of the param, but here we need to?

**Review by bernt-matthias — APPROVED** [review]

File: tools/scpipe/scpipe.xml (hunk 2 above)
Comment (mvdbeek) [reply tools/scpipe/scpipe.xml:247]: Right, if you don't add an ftype the format guessed at upload will be chosen. IF we do specify it it need to be correct though, so we shouldn't use `ftype="fasta"` when the file is gzipped.

Comment (bernt-matthias) [reply tools/scpipe/scpipe.xml:247]: thanks for the explanation

**Pull request merged**